src/Microsoft.AspNetCore.Routing.Abstractions/IParameterTransformer.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public interface IParameterTransformer : IParameterPolicy
```

Comment:
Needs docs. We should likely explain that this is uni-directional, and only applies to static parameter replacement.